src/index.html

```diff
@@ -11,7 +11,132 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <div class="form-field">
+        Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
```

Comment:
```suggestion
          Surname:
          <input
            type="text"
            name="surname"
            autocomplete="off"
          >
```
these elements are nested at the same level